changelog/2021/MAY.rst

```diff
@@ -0,0 +1,7 @@
+--------------------------------------------------------------------------------
+                                New
+--------------------------------------------------------------------------------
+* IOSXE
+    * Added ShowIpNbarVersion:
+		* Add Show command 'show ip nbar version'
+		* Add folder based unittests
```

Comment:
Please rename the changelog file and move into `changelog/undistributed`

Reply:
Fixed!